package/src/Luna/Package/Env.hs

```diff
@@ -0,0 +1,32 @@
+module Luna.Package.Env where
+
+import Prologue
+
+import qualified Data.Char          as Char
+import qualified System.Environment as Env
+
+import Luna.IR (Name)
+
+packageEnvVarPrefix :: String
+packageEnvVarPrefix = "LUNA_LIBRARY_"
+
+-- | This function translates a name like MyLibrary to an underscored
+--   environment variable name: LUNA_LIBRARY_MY_LIBRARY.
+packageNameToEnvVarName :: Text -> Text
+packageNameToEnvVarName txt = convert $ packageEnvVarPrefix <> envName where
+    envName = Char.toUpper <$> intercalate "_" wordsByCapitalization
+    wordsByCapitalization = reverse $ split "" (convert txt)
+    split []       []             = []
+    split prevWord []             = [reverse prevWord]
+    split prevWord (char : chars) = if Char.isUpper char
+        then let
+            recur = split [char] chars
+            in if null prevWord then recur else reverse prevWord : recur
+        else split (char : prevWord) chars
+
+setLibraryVar :: MonadIO m => Name -> FilePath -> m ()
+setLibraryVar name path = liftIO $ Env.setEnv varName path where
```

Comment:
Do we want to be able to override these paths by setting them externally? If so, you'd need to check if they already exist.

Reply:
Not in this way, no. We may want to enable overrides for library locations, but for the whole library, not just this way of exposing it. Running code from one location but exposing another sounds like a nonstarter.